package.json

```diff
@@ -4,8 +4,14 @@
   "version": "0.1.5-beta",
   "author": "Teamleader <development@teamleader.eu>",
   "betterScripts": {
+    "storybook": {
+      "command": "start-storybook -p 3000",
```

Comment:
Is it possible to use the port of the config? I don't want to close my stuff running on 3000

Reply:
You can now use `yarn start -- --port 1337` to change the port.